src/commands/build.js

```diff
@@ -29,6 +29,7 @@ export default (program: any) => {
     .option('-t --type <build>', 'Type of build: [archive|simulator].', /^(archive|simulator)$/i)
     .option('--release-channel <channel-name>', 'Pull from specified release channel.', 'default')
     .option('--no-publish', 'Disable automatic publishing before building.')
+    .option('--use-ci', 'CI Credentials store in env')
```

Comment:
I think instead of a command line flag, it would be good to just implicitly pull these from environment variables if they're all set. Something like:

```js
if (process.env.ANDROID_KEYSTORE_PATH && process.env.ANDROID_KEYSTORE_ALIAS && ...) {
  creds = { process.env... };
} else {
  creds = promptUser();
}
```

What do you think about that?

Reply:
done